.github/renovate.json

```diff
@@ -4,9 +4,9 @@
     ":preserveSemverRanges"
   ],
   "labels": ["auto-update"],
+  "dependencyDashboardAutoclose": true,
```

Comment:
Why is this changed?

Reply:
I am not 100% sure, but belongs to commit done by @cloudpossebot : https://github.com/cloudposse/terraform-aws-ecr-public/pull/4/commits/932a629161b70318f05e46ca1ba392adcaaf9ad2